pkg/dispatchcli/cmd/install.go

```diff
@@ -1088,6 +1101,7 @@ func runInstall(out, errOut io.Writer, cmd *cobra.Command, args []string) error
 			"global.registry.uri":                                 config.DispatchConfig.ImageRegistry.Name,
 			"global.registry.insecure":                            strconv.FormatBool(config.DispatchConfig.ImageRegistry.Insecure),
 			"global.tls.secretName":                               config.DispatchConfig.TLS.SecretName,
+			"global.zookeeper.location":                           config.Zookeeper.Location,
```

Comment:
`config.Zookeeper.Location` needs to be changed when installing in the single namespace mode. This is similar to what we did for kafka brokers in https://github.com/vmware/dispatch/blob/9120bdd5b43cc6eebdfd9c6a23a67e95e986ebbb/pkg/dispatchcli/cmd/install.go#L794